bindings/python/py_src/gtars/refget/__init__.pyi

```diff
@@ -0,0 +1,109 @@
+from typing import Union, Optional, List, Type
+from enum import Enum
+from os import PathLike
+
+class AlphabetType(Enum):
+    """
+    Represents the type of alphabet for a sequence.
+    """
+    Dna2bit: int
+    Dna3bit: int
+    DnaIupac: int
+    Protein: int
+    Ascii: int
+    Unknown: int
+
+    def __str__(self) -> str: ...
+
+class SequenceMetadata:
+    """
+    Metadata for a biological sequence.
+    """
+    name: str
+    length: int
+    sha512t24u: str
+    md5: str
+    alphabet: AlphabetType
+
+    def __repr__(self) -> str: ...
+    def __str__(self) -> str: ...
+
+class SequenceRecord:
+    """
+    A record representing a biological sequence, including its metadata and optional data.
+    """
+    metadata: SequenceMetadata
+    data: Optional[bytes] # Vec<u8> maps to bytes in Python
+
+    def __repr__(self) -> str: ...
+    def __str__(self) -> str: ...
+
+class SeqColDigestLvl1:
+    """
+    Level 1 digests for a sequence collection.
+    """
+    sequences_digest: str
+    names_digest: str
+    lengths_digest: str
+
+    def __repr__(self) -> str: ...
+    def __str__(self) -> str: ...
+
+class SequenceCollection:
+    """
+    A collection of biological sequences.
+    """
+    sequences: List[SequenceRecord]
+    digest: str
+    lvl1: SeqColDigestLvl1
+    file_path: Optional[str]
+    has_data: bool
+
+    def __repr__(self) -> str: ...
+    def __str__(self) -> str: ...
+
+class StorageMode(Enum):
+    """
+    Defines how sequence data is stored in the Refget store.
+    """
+    Raw: int
+    Encoded: int
+
+class GlobalRefgetStore:
+    """
+    A global store for refget sequences, allowing import, retrieval, and storage operations.
+    """
+    def __init__(self, mode: StorageMode) -> None: ...
+    def import_fasta(self, file_path: Union[str, PathLike]) -> None: ...
+    def get_sequence_by_id(self, digest: str) -> Optional[SequenceRecord]:
+        """
+        Retrieves a sequence record by its SHA512t24u or MD5 digest.
+        """
+        ...
+    def get_sequence_by_collection_and_name(self, collection_digest: str, sequence_name: str) -> Optional[SequenceRecord]: ...
+    def get_substring(self, seq_digest: str, start: int, end: int) -> Optional[str]: ...
+    def write_store_to_directory(self, root_path: Union[str, PathLike], seqdata_path_template: str) -> None: ...
+
+    @classmethod
+    def load_from_directory(cls: Type["GlobalRefgetStore"], root_path: Union[str, PathLike]) -> "GlobalRefgetStore": ...
```

Comment:
Maybe consider adding some more docstrings to these? Unless they are self-explanatory to a user.
```python
def import_fasta(self, file_path: Union[str, PathLike]) -> None: ...
    """
    Import a new fasta sequence from disk
    
    Args:
        file_path - str - the path to import from
    """
```